README.md

```diff
@@ -1,6 +1,6 @@
 # Asynchronous JavaScript Client for the Giant Swarm API
 
-For details about the Giant Swarm API, please check out the [API documentation](https://docs.giantswarm.io/reference/api/).
+For details about the Giant Swarm API, please check out the [API documentation](https://docs.giantswarm.io/api).
```

Comment:
Tiny: the actual URL ends with a `/`

Reply:
Personally I dont like trailing slashes, but I can fix it (I see works doing a redirect)